src/boot/efi/pe.c

```diff
@@ -307,17 +309,17 @@ EFI_STATUS pe_kernel_info(const void *base, uint32_t *ret_compat_address) {
 }
 
 EFI_STATUS pe_memory_locate_sections(
-                const void *base,
-                const char* const sections[],
-                PeSectionVector *ret_sections) {
+                const void* base,
```

Comment:
Hmm?

Reply:
I thought you liked that style better? You guys keep shifting around he asterisk left to right or right to left, I though you care about that, and if I touch this I should fix this?

I personally really don't care whether the asterisk is placed before or after the space, I am just trying to avoid any discussion about this. Seems I failed though.

Hence, maybe I am misunderstanding you, but what if this is not about where asterisk and here space is placed, what is this about?

If you have an opinion where the asterisk and where the space on these things shall be placed, maybe formalize it in CODING_STYLE.md, because it right now does it sometimes one way, sometimes the other, because I wrote about it, and I really don't care.

Reply:
So the current style is: always use `const char*` for function return types, but `const char *var` for params/variable definitions. The reason is that the function return type can be seen as whole, which is nicer.

Reply:
uh, that's a weird style though, i.e. differently at two different places?

but if this is what everyone wants I am fine with it, but could you please add this to CODING_STYLE.md?

Reply:
-> #33578